### ISSUE TITLE
Obscene Burn Damage Needs a Solution

### ISSUE THREAD

**Issue by Admiral-Obvious-001** [space-wizards/space-station-14#25252]
## Description

This recently happened on a Salamander shift where a sentient artifact managed to get inside the walls and spew tritium gas on demand. This turned into a major plasma fire that ended up killing a lot of the crew, most patients were at over 2k burn damage and rising shortly after the fire.

**Reproduction**

Start a plasma or tritium fire and stand within with minimal/no protection, you will crit rapidly and then continue to burn to death. After death your body temperature will likely stay absurdly high and continue to apply heat damage. This effectively round removes you from the game but also doesn't. Med usually can't deal with this without absurd amounts of necrosol or cloning.

**Additional context**

We were discussing this on Discord and suggested that excessive heat/shock damage turns a player to ash once exceeding a certain large value, such as over 500 heat. Additionally there needs to be a more effective mechanic to reduce body heat of an afflicted individual besides just dumping them into cyro which is extremely ineffective.

**Comment by IProduceWidgets** [comment]
There is leporazine for an easy temp fix. That's probably the easiest. Other chemicals will cool you, but they're more finicky.

**Comment by Kadeo64** [comment]
I don't think that completely dusting a player is a good option with high heat damage. just port husks from ss13 when you have over 300+ burn damage and require an unhusking cryo chem

**Comment by deltanedas** [comment]
not to mention 5k burn victims can be borged, ash just round removes them

**Comment by forgotmyotheraccount** [comment]
Instead of ashing burn victims, why not reduce them to a skeleton? It's more comedic and would let you preserve the brain inside the skull (crack them open with the circular saw or something). I do hope we eventually get more gibbing alternatives, not just for burn damage but for other damage types too. Take enough radiation damage and you melt into a puddle of flesh and organs.

**Comment by Peptide90** [comment]
We have support for this in a way.

Destruction conditions are used to not spawn debris if something sustains massive damage at once. Presume you can set it per damage type for heat, radiation, etc and have them turn to goo. Preserving the brain and character might need something else though.

**Comment by deltanedas** [comment]
see also exterminator uses this to have special gib/burn damage and popups

**Comment by Admiral-Obvious-001** [comment]
The main thing I want to bring attention to this issue is that burn damage will functionally never stop and med simply can't deal with it.

**Comment by Admiral-Obvious-001** [comment]
![image](https://github.com/space-wizards/space-station-14/assets/89495925/04c2ce87-9165-499b-920a-b8988e680149)
Another fun example of this issue with everyone's favorite fox, but on the other end of the temperature spectrum.

**Comment by K-Dynamic** [comment]
If excessive heat turns your corpse to ash/husk, excessive cold should turn your corpse into an ice block

Both should be capped idk

**Comment by Peptide90** [comment]
```
 - type: TransferMindOnGib
  - type: Destructible
    thresholds:
    - trigger:
        !type:DamageTypeTrigger
        damageType: Heat
        damage: 200
      behaviors:
      - !type:GibBehavior
```
Then you'll want to add a spawn behaviour in with that gib behaviour for your husk as well as this to your husk entity:
```  - type: Tag
    tags:
    - MindTransferTarget```
Only issue is I'm not sure how it'll transfer all your stuff over. 

So first thing is you can use the above for gibbing if you don't want them to be in the round anymore. But an actual solution would just be using mob visualiser states for being too burnt or frozen in a block of ice, and setting a limit to the amount of damage someone can take per type / subtype. So they cap out at like -250 damage or gib at that point. Depends what people want for balance whether they want it to gib or just cap.

**Comment by LordEclipse** [comment]
I think capping the damage and having husk/ice block is preferable, since it offers a way to actually counterplay spaced bodies should you find them, or excessively burnt bodies, otherwise it just sucks.

**Comment by QuietlyWhisper** [comment]
- This may or may not have been resolved by #26971
